client/src/components/form/UpdateBiodata.tsx

```
@@ -1,160 +1,188 @@
-              <button disabled={activeStep === 0} onClick={handleBack} className="formbtn">
+            {error && (
+              <span className="text-red-800 border border-red-900 bg-red-100 text-center font-semibold rounded p-2 my-2">
+                {error}
+              </span>
+            )}
+            <div className="flex flex-row items-center justify-between m-2 mx-6">
+              <button
+                disabled={activeStep === 0 || isLoading}
+                onClick={handleBack}
+                className="formbtn"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)